src/components/AccountSettings.tsx

```diff
@@ -185,8 +186,8 @@ const AccountSettings = (props: AccountSettingsProps) => {
                             <>
                                 <div className='label-cell'>OrcID: </div>
                                 <div className='orcid-cell'>
```

Comment:
This should use orcid from the component state (which is set based on the UserBundle).  If an orcid is undefined, then show what it used to (a Bind ORCiD button).  If set then show your new edit button.